### PR TITLE
Nerpa update

### PR DESCRIPTION
**Pull request by mishmosh**

### CODE REVIEW

**Comment by travisperson** [comment]
@mishmosh https://github.com/filecoin-project/network-info/pull/7

File: docs/how-to/networks.md [hunk 1]
```diff
@@ -102,16 +102,16 @@ Nerpa is a long-lived devnet, **best for developers building storage apps**. Ner
 | --- | --- |
 |Sector Sizes | 512MiB |
 |Branch | `ntwk-nerpa` |
-|Tag | [`tag ntwk-nerpa-7.7.0`](https://github.com/filecoin-project/lotus/tree/ntwk-nerpa-7.7.0) |
-|Details page | [nerpa.fildev.network](http://www.nerpa.fildev.network/) |
-|Next expected reset | August 12, 2020 |
+|Tag | n/a |
+|Details page | n/a |
```
Comment (eshon) [inline docs/how-to/networks.md:106]: Once Travis' [network-info PR 7 is merged](https://github.com/filecoin-project/network-info/pull/7/files) we can point it to https://network.filecoin.io/ ?

Comment (mishmosh) [reply docs/how-to/networks.md:106]: The PR is merged but nerpa info doesn't appear to show up on the website. I only see the details for calibration.

**Review by eshon — COMMENTED** [review]
@pooja do you know how to trigger an update to network.filecoin.io?

**Comment by eshon** [comment]
Also should https://stats.nerpa.interplanetary.dev/ be added here?

**Comment by mishmosh** [comment]
> Also should https://stats.nerpa.interplanetary.dev/ be added here?

Yes! Now that it's live, just added.

**Comment by mishmosh** [comment]
Merging now, can add network.filecoin.io in a follow-up PR once that's updated.

**Comment by pooja** [comment]
If you go to the Filecoin network link, you can see it has already been updated. It redeploys every time there's a new merge to master in the network info repo.

**Pull request merged**